sdk/keyvault/azure-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyAsyncClientTest.java

```diff
@@ -39,7 +39,7 @@ protected void beforeTest() {
                 .pipeline(pipeline)
                 .endpoint(getEndpoint())
                 .buildAsyncClient());
-        }
```

Comment:
This change can be reverted :-)

Reply:
👍

Reply:
It was added to test SDK with PagedIterable change

Reply:
removed now.